CHANGELOG

```diff
@@ -8,6 +8,24 @@ INIFileEditor - graphical parameter editor for INI files
 Parameters - list of algorithm or TOPP tool parameters that changed in this release
 
 ------------------------------------------------------------------------------------------
+----                                OpenMS 2.1                                        ----
+------------------------------------------------------------------------------------------
+OpenMS 2.1 is the first release that provides an notification mechanism which 
```

Comment:
`.. an update notification functionality` (instead of 'notification mechanism')??